src/helpers/cliUtils.js

```diff
@@ -149,17 +187,16 @@ async function app(Client, fromDate, toDate, pathToConfig, pathToRunLogs, debug,
 
     logger.info(`Extracting data for ${patientIds.length} patients`);
     errors = await extractDataForPatients(auth, config, patientIds, icareClient, messagingClient, runLogger, effectiveFromDate, toDate);
+
+    const { notificationInfo } = config;
+    if (notificationInfo) {
+      await sendEmailNotification(notificationInfo, errors);
+    }
   } catch (e) {
     logger.error(e.message);
     logger.debug(e.stack);
     process.exit(1);
   }
-
-  // log errors for each patient
-  Object.keys(errors).forEach((index) => {
-    logger.debug(`Errors for patient at row ${parseInt(index, 10) + 1} in .csv file:`);
-    errors[index].forEach((e) => logger.debug(e.message.trim()));
-  });
```

Comment:
DP4 - Don't feel strongly, but could we maybe leave this in? Thought it was nice to see the errors aggregated like this. If we do, we should use the logger.error level? We probably shouldn't be logging errors at the debug level? Open to thoughts here

Reply:
I don't feel too strongly either way, but I think I initially logged it at the debug level because the actually errors are still being logged(at the error level) during execution and decided to log the "summary" of errors at the end of execution in debug mode.

Reply:
Hm yeah I had removed this because I thought the summary was here as a place holder until we had the email, and I felt like it was a little redundant with the actual errors being logged above and the email. But if you feel strongly about it I can put it back in or we can talk about it further.

Reply:
Let's leave it out 😄